test/table.test.js

```diff
@@ -1,35 +1,135 @@
-const createTestTable = require('./table');
+const createTestTable = require("./table");
 
 // toBe for primitives like strings, numbers or booleans for everything else use toEqual(object)
 
-test('Alpha - Capitalized ', () => {
-  expect(createTestTable(['Echo','Alpha','Bravo','Charlie','Delta'])).toStrictEqual(['Alpha','Bravo','Charlie','Delta','Echo']);
+test("Alpha - Capitalized ", () => {
```

Comment:
Changing to the vertical format makes sense. I don't have experience with setting up tests, well I guess I do now. :test_tube: